docs/api/plugins.rst

```diff
@@ -1,39 +1,22 @@
 Plugins
 ========
 
-`User docs: Installing a Plugin </docs/plugins/>`__
 
 Plugin file structure
 ----------------------
 
 Most features can be self-contained in their own .py file, so that the feature is enabled or disabled by ``import``-ing that .py file or not.
 A plugin is usually a single .py file, installed on the same computer as visidata.
-Plugins can be installed and uninstalled in the :ref:`Plugins Sheet </docs/plugins>`.
+`Instructions on how to install a plugin </docs/plugins/>`__
 
 Plugins often start as a small snippet in ``.visidatarc``, and then the code is migrated to a separate file when it gets too large to share with other people via a short code snippet.
 The actual code in either case should be identical.
 
-To publish a plugin, create a public repo with a .py file. Ensure the plugin has a ``__version__``. In the `plugins.jsonl file <https://raw.githubusercontent.com/visidata/dlc/stable/plugins.jsonl>`__ in the `visidata:dlc repo <https://github.com/visidata/dlc>`__ , add a row for each plugin with all of the necessary information:
-
-- *name*: short name of the plugin (like ``vfake``).  Less than 20 characters.
-- *description*: a "one line" searchable description of the core features.  Less than 1000 characters.
-- *maintainer*: like ``Your Name <name@example.com>``.
-- *latest_release*: date of most recent release, ISO formatted like ``2020-02-02``.
-- *latest_ver*: version of most recent release, like ``v1.4``.
-- *url*: link to the primary page (which may be the raw .py file itself, if it describes itself effectively).
-- *visidata_ver*: version of VisiData required, like ``v2.0``.
-- *pydeps*: space-separated list of PyPI dependencies (like in ``requirements.txt``).
-- *vdplugindeps*: space-separated list of vd plugin dependencies.
-- *sha256*: SHA256 hash of plugin .py of most recent release. A script for obtaining this has can be found `in dev/vdhash.py <https://raw.githubusercontent.com/saulpw/visidata/develop/dev/vdhash.py>`__ .
-
 .. note::
 
-    VisiData installs plugins in the ``$HOME/.visidata/plugins/`` directory.
-
-    ``import`` statements for all enabled plugins go into ``$HOME/.visidata/plugins/__init__.py``.
+    At startup, all .py files placed in the ``$HOME/.visidata/`` directory or Python code added to a ``~/.visidatarc`` will be auto-imported into VisiData.
```

Comment:
It's not true that all .py files in .visidata will be imported.  The `.visidata/plugins` module will be imported only (and thus imported modules in its `__init__.py`).

Reply:
I got muddled there; good catch!